addons/sms/static/src/messaging/component/message/message_tests.js

```diff
@@ -0,0 +1,172 @@
+odoo.define('sms.messaging.component.MessageTests', function (require) {
+'use strict';
+
+const components = {
+    Message: require('mail.messaging.component.Message'),
+};
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    pause,
+    start: utilsStart,
+} = require('mail.messaging.testUtils');
+
+QUnit.module('sms', {}, function () {
+QUnit.module('messaging', {}, function () {
+QUnit.module('component', {}, function () {
+QUnit.module('Message', {
+    beforeEach() {
+        utilsBeforeEach(this);
+        this.createMessageComponent = async message => {
+            const MessageComponent = components.Message;
+            MessageComponent.env = this.env;
+            this.component = new MessageComponent(null, {
+                messageLocalId: message.localId,
+            });
+            delete MessageComponent.env;
+            await this.component.mount(this.widget.el);
+        };
+        this.start = async params => {
+            let { env, widget } = await utilsStart(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        utilsAfterEach(this);
+        if (this.component) {
+            // The component must be destroyed before the widget, because the
+            // widget might destroy the messaging service before destroying the
+            // component, and the Message component is relying on messaging.
+            this.component.destroy();
+        }
+        if (this.widget) {
+            this.widget.destroy();
+        }
+        this.env = undefined;
+    },
+});
+
+QUnit.test('Notification Sent', async function (assert) {
+    assert.expect(8);
+
+    await this.start();
+    const message = this.env.entities.Message.create({
+        id: 10,
+        message_type: 'sms',
+        notifications: [['insert', [{
+            id: 11,
+            notification_status: 'sent',
+            notification_type: 'sms',
+            partner: [['insert', { id: 12, name: "Someone" }]],
+        }]]],
+    });
+    await this.createMessageComponent(message);
+
+    assert.containsOnce(
+        document.body,
+        '.o_Message',
+        "should display a message component"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_Message_notificationIcon',
+        "should display the notification icon"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_Message_notificationIcon .fa-mobile',
```

Comment:
same comment as before: `<i>` should contain `notificationIcon` so that `fa-*` assertion is on icon itself.